src/main/java/com/easypost/model/Pickup.java

```diff
@@ -17,4 +22,44 @@ public final class Pickup extends EasyPostResource {
     private Address address;
     private List<CarrierAccount> carrierAccounts;
     private List<PickupRate> pickupRates;
+
+    /**
+     * Get the lowest rate for this Pickup.
+     *
+     * @return lowest PickupRate object
+     * @throws EasyPostException when the request fails.
+     */
+    public PickupRate lowestRate() throws EasyPostException {
+        return this.lowestRate(null, null);
+    }
+
+    /**
+     * Get the lowest rate for this Pickup.
+     *
+     * @param carriers The carriers to use in the filter.
+     * @param services The services to use in the filter.
+     * @return lowest PickupRate object
+     * @throws EasyPostException when the request fails.
+     */
+    public PickupRate lowestRate(final List<String> carriers, final List<String> services)
```

Comment:
Same about base-overload order here

Reply:
Sure, I will adjust this in the next PR so I don't dismiss approvals.